src/main/java/uk/gov/companieshouse/pscstatement/delta/consumer/PscStatementDeltaConsumer.java

```diff
@@ -49,11 +51,16 @@ public PscStatementDeltaConsumer(PscStatementDeltaProcessor deltaProcessor,
     public void receiveMainMessages(Message<ChsDelta> message,
                                     @Header(KafkaHeaders.RECEIVED_TOPIC) String topic,
                                     @Header(KafkaHeaders.RECEIVED_PARTITION_ID) String partition,
-                                    @Header(KafkaHeaders.OFFSET) String offset) {
+                                    @Header(KafkaHeaders.OFFSET) String offset) throws IOException,
```

Comment:
Should our consumer be throwing exceptions? I'd prefer us to have caught and handled as retry/non-retryables